### PR TITLE
Make sure examples compile

### PR DESCRIPTION
**Pull request by diyessi**

### CODE REVIEW

File: doc/examples/CMakeLists.txt [hunk 1]
```diff
@@ -17,6 +17,7 @@
 if (NGRAPH_CPU_ENABLE)
```
Comment (silee2) [inline doc/examples/CMakeLists.txt]: Any reason why we only build examples if cpu backend is enabled? Interpreter too slow for mnist_mlp?

**Comment by indie** [comment]
Review / fixes on PR #3995

File: doc/examples/CMakeLists.txt (hunk 1 above)
Comment (diyessi) [reply doc/examples/CMakeLists.txt]: Maybe it was to make sure they built once.

**Comment by silee2** [comment]
Please run `style-apply`

**Pull request merged**